docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
+data needs:
+
+#. The probability that a simulant will start the simulation in a given state
+   within the cause model.
+#. The probability that a simulant will transition to a new state within the
+   cause model in a given time-step.
+#. The disability weight for each state in the cause model
+#. The probability that a simulant in a given cause model state will die in a
+   given time-step.
+
+There are several common data sources that can be used for these needs, which
+are outlined in the table below and discussed in more detail afterward.
+`Non-standard data sources`_ are discussed later on this page.
+
+.. list-table:: Data Definitions
+   :widths: 20 30 30
+   :header-rows: 1
+
+   * - Measure
+     - Definition
+     - Uses
+   * - `Prevalence`_
+     - Proportion of population with a given condition.
+     - Initialize cause model states.
+   * - `Birth Prevalence`_
+     - Proportion of all live births born with a given condition.
+     - Initialize neonatal cause model states.
+   * - `Incidence`_
+     - Number of new cases of a given condition per person-year.
+     - Estimate transition rates.
+   * - `Remission`_
+     - Number of recovered cases from a given condition per person-year.
+     - Estimate transition rates.
+   * - `Disability Weights`_
+     - Proportion of full health not experienced due to disability associated
+       with a given condition.
+     - Measure disability attributed to cause model states.
+   * - `Cause-specific Mortality`_
+     -
+     -
+   * - `Excess Mortality`_
+     -
+     -
 
-Incidence
-+++++++++
+Prevalence
+++++++++++
+
+Prevalence is defined as the **proportion of a given population that possesses
+a given condition or trait** at a given time-point.
 
-Birth prevalence
+  For example, the prevalence of obesity in the United States was
+  approximately 40% in 2016.
+
+When a *time-frame* (such as 2016, i.e. 1/1/16 to 12/31/16) instead of a
+*time-point* (such as 1/1/16) is reported, it is commonly assumed that the
+reported prevalence represents the prevalence of the *midpoint* of
+that time-frame (7/1/16 is the midpoint for the time frame of all of 2016).
+However, this may not always be the case and it should be considered when
+relevant.
+
+Prevalence data can be used to **initialize cause model states** and
+represents the **probability that a simulant will begin the simulation in a
+given state.**
+
+  For example, the probability that a simulant in a model of obesity in the
+  United States beginning in 2016 will begin the simulation as obese is 0.4 or
+  40%.
+
+Birth Prevalence
 ++++++++++++++++
 
-Remission
+Birth prevalence is defined as the **proportion of live births in a given
+population that possess a given condition or trait at birth.**
+
+  For example, the birth prevalence for cleft lip in the United States in 2006
+  was 10.6 per 10,000 live births, or 0.106%.
+
+Birth prevalence data can be used to **initialize neonatal cause model
```

Comment:
Prevalence is used to initialize states for the starting pop, birth prevalence for sims who enter by being born in the simulation.

Reply:
I added some clarification here